taxcalc/functions.py

```diff
@@ -878,26 +873,33 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517, e37717,
     # imputation for x62730
     if e62730 != 0 and e24515 > 0:
         x62730 = e62730 - e24515
-    c62730 = e24515 + x62730
+    c62730 = e24515
 
     _amtded = c60200 + c60220 + c60240
-    _prefded = c60200 + c60220 + c60240
+    if FDED == 2:
+        _prefded = 0
+    else:
+        _prefded = c60200 + c60220 + c60240
     _prefnot = c21060 - c21040 - _prefded
     _totded = _prefded + _prefnot
     _useded = min(_totded, max(0, c00100 - c04600))
-    c04500 = c00100 - max(_useded, _standard + e37717)
+    # c04500 = c00100 - max(_useded, _standard + e37717)
     if FDED == 1:
-        c60000 = c00100 - _useded
+        c60000 = c00100 - _totded
+    elif FDED == 3:
+        c60000 = c00100 - max(c04470, t04470)
     else:
         c60000 = c00100
-    c60000 = c00100 - _useded
+
+    c60000 = c60000 - e04805
+    # c60000 = c00100 - _useded
     _amtded = min(_prefded, max(0, _useded - _prefnot))
     # if c60000 <= 0:
 
     #   _amtded = max(0., _amtded + c60000)
 
-    if FDED == 1 or ((_amtded + e60290) > 0):
-        _addamt = _amtded + e60290 - c60130
+    if FDED == 1 or ((_prefded + e60290) > 0):
```

Comment:
`_amtded` was equivalent to `_prefded` before. And now `_prefded` is zero when `FDED == 2`. Thus the only scenario affected here was when `FDED ==2`, but I'm not quite sure about why we are making the change here.